magenta_docker/README.md

```diff
@@ -0,0 +1,89 @@
+# DDSP Docker
+
+Docker image for training autoencoder on [Google Cloud AI Platform](https://cloud.google.com/ai-platform).
+
+### Before you begin
+Make sure that you have completed the following steps:
+* Set up your [GCP project](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
+* Create a [Google Cloud Storage Bucket](https://cloud.google.com/storage/docs/creating-buckets)
+* Enable [AI Platform Training and Prediction, Container Registry, and Compute Engine APIs](https://pantheon.corp.google.com/flows/enableapi?apiid=ml.googleapis.com,compute_component,containerregistry.googleapis.com)
+* Install [Docker](https://docs.docker.com/engine/install/)
+* [Configure Docker for Cloud Container Registry](https://cloud.google.com/container-registry/docs/pushing-and-pulling)
+* [Upload the training data](https://cloud.google.com/storage/docs/uploading-objects) in the [TFRecord](https://www.tensorflow.org/tutorials/load_data/tfrecord) format to the storage bucket. You can preprocess your audio files into this format using the `ddsp_prepare_tfrecord` tool as described in [Making a TFRecord dataset from your own sounds](https://github.com/magenta/ddsp/tree/master/ddsp/training/data_preparation).
+
+### Quickstart:
+
+#### Define some environment variables
+
+If you are located outside Europe we recommend setting `$REGION` accordingly to your location. We also recommend to [setup hostname](https://cloud.google.com/container-registry/docs/pushing-and-pulling#tag_the_local_image_with_the_registry_name) in `$IMAGE_URI` based of the `$REGION` choice as if your Docker images are stored in different region than the job is computed additional charges will be applied. 
```

Comment:
I think this readme should be region agnostic. In any case use the default us-central1 default region location just for explainability.

Reply:
Done